test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeUnsafeTestClasses.java

```diff
@@ -48,4 +51,12 @@
 			this.y = y;
 		}
 	}
+
+	static primitive class ThisClassWillHavePaddingWhenReferencesAreCompressed {
```

Comment:
You can change this class name to sth like `ValueTypeWithLongField`. Or make a class like `ValueTypePoint2DLong` which has 2 `ValueTypeLong` fields.

Reply:
addressed in 780435c